template/lookup/lookup.html

```diff
@@ -7,6 +7,7 @@
            typeahead-on-select="onSelect($item, $model, $label)"
            typeahead-editable="false"
```

Comment:
Add typehead debounce option for 500ms to improve remote data querying performance